express-notes-api/server.js

```diff
@@ -0,0 +1,109 @@
+import express from 'express';
+
+import { readFile, writeFile } from 'node:fs/promises';
+
+const app = express();
+const json = JSON.parse(await readFile('data.json'));
+let message = {};
+
+app.use(express.json());
+
+app.get('/api/notes', (req, res) => {
+  try {
+    res.json(Object.values(json.notes));
+    res.status(200);
```

Comment:
It's more common to chain these: `res.status().json()`. Like you did below when sending a message.

Also, 200 is the default so you don't actually even need this call.